plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTablePqlExtractor.java

```diff
@@ -83,6 +84,16 @@ public static String extractPql(DynamicTable table, TupleDomain<ColumnHandle> tu
             }
             builder.append(table.getLimit().getAsLong());
         }
+        if (!table.getQueryOptions().isEmpty()) {
+            builder.append(";\n");
+            for (Map.Entry<String, String> queryOption : table.getQueryOptions().entrySet()) {
+                builder.append("SET ")
+                        .append(queryOption.getKey())
+                        .append(" = ")
+                        .append(queryOption.getValue())
```

Comment:
The official Pinot documentation contains single-quotes for string literal. Is there no need to quote for such cases? 
```sql
SET key1 = 'value1';
SET key2 = 123;
SELECT * FROM myTable
```

Reply:
Yes, updated with single quotes.